spec/rubocop/cop/sorbet/scope_helper_position_spec.rb

```diff
@@ -0,0 +1,237 @@
+
+          configure do
+            something
+          end
+
+          sealed!
+          ^^^^^^^ Cannot invoke sealed! after method definitions or invocations
+
+          sig { abstract.void }
+          def bar; end
+        end
+      RUBY
+    end
+
+    it("autocorrects moving the invocation to the right spot") do
+      source = <<~RUBY
+        module Interface
+          extend T::Sig
+          extend T::Helpers
+
+          sig { abstract.void }
+          def foo; end
+
+          final!
+
+          sig { abstract.void }
+          def bar; end
+        end
+      RUBY
+
+      corrected_source = <<~CORRECTED
+        module Interface
+          extend T::Sig
+          extend T::Helpers
+
+          final!
+
+          sig { abstract.void }
+          def foo; end
+
+          sig { abstract.void }
+          def bar; end
+        end
+      CORRECTED
+
+      expect(autocorrect_source(source)).to(eq(corrected_source))
+    end
+
+    it("autocorrects when there are no spaces after method definition") do
+      source = <<~RUBY
+        module Interface
+          extend T::Sig
+          extend T::Helpers
+
+          sig { abstract.void }
+          def foo; end
+          final!
+
+          sig { abstract.void }
+          def bar; end
+        end
+      RUBY
+
+      corrected_source = <<~CORRECTED
+        module Interface
+          extend T::Sig
+          extend T::Helpers
+
+          final!
+
+          sig { abstract.void }
+          def foo; end
+
+          sig { abstract.void }
+          def bar; end
+        end
+      CORRECTED
+
+      expect(autocorrect_source(source)).to(eq(corrected_source))
+    end
+
+    it("autocorrects when there are no spaces before method definition") do
+      source = <<~RUBY
+        module Interface
+          extend T::Sig
+          extend T::Helpers
+
+          sig { abstract.void }
+          def foo; end
+
+          final!
+          sig { abstract.void }
+          def bar; end
+        end
+      RUBY
+
+      corrected_source = <<~CORRECTED
+        module Interface
+          extend T::Sig
+          extend T::Helpers
+
+          final!
+
+          sig { abstract.void }
+          def foo; end
+
+          sig { abstract.void }
+          def bar; end
+        end
+      CORRECTED
+
+      expect(autocorrect_source(source)).to(eq(corrected_source))
+    end
+
+    it("autocorrects when there are no signatures") do
+      source = <<~RUBY
+        class Final
+          extend T::Sig
+          extend T::Helpers
+
+          def foo; end
+
+          final!
+
+          def bar; end
+        end
+      RUBY
+
+      corrected_source = <<~CORRECTED
+        class Final
+          extend T::Sig
+          extend T::Helpers
+
+          final!
+
+          def foo; end
+
+          def bar; end
+        end
+      CORRECTED
+
+      expect(autocorrect_source(source)).to(eq(corrected_source))
+    end
+
+    it("autocorrects when there are method invocations") do
```

Comment:
Can we add a test with all the helpers in the same scope to ensure we don't create an infinite loop in a later change? 🙏 

```rb
class Foo
  extend T::Helpers

  def foo; end

  interface!
  abstract!
  sealed!
  final!
end
```

Reply:
As for maximum coverage we might also want a test when there is no `include/extend/prepend`:

```rb
class Foo
  def  foo; end # with or without this

  interface! # and maybe 
  abstract! # a few inlined
  sealed! # comments
  final! # to ensure the right behaviour
end
```